powerline/bindings/lemonbar/powerline-lemonbar.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+import sys
+import time
+import re
+import subprocess
+
+from threading import Lock, Timer
+from argparse import ArgumentParser, REMAINDER
+
+from powerline.lemonbar import LemonbarPowerline
+from powerline.lib.monotonic import monotonic
+from powerline.lib.dict import mergedicts
+
+
+if __name__ == '__main__':
+	parser = ArgumentParser(
+		description='Powerline BAR bindings.',
+		usage='%(prog)s [-h] [--i3] [--height HEIGHT] [--interval INTERVAL] -- args')
+	parser.add_argument(
+		'--i3', action='store_true',
+		help='Subscribe for i3 events.'
+	)
+	parser.add_argument(
+		'--height', default='',
+		help='Bar height'
+	)
+	parser.add_argument(
+		'--interval', '-i',
+		default=0.5,
+		help='Refresh interval'
+	)
+	parser.add_argument(
+		'args', nargs=REMAINDER,
+		help='Extra arguments for lemonbar'
+	)
+	args = parser.parse_args()
+
+	active_screens = [match.groupdict() for match in re.finditer(
+		'^(?P<name>\w+) connected (?P<width>\d+)x(?P<height>\d+)\+(?P<x>\d+)\+(?P<y>\d+)',
+		subprocess.check_output(['xrandr', '-q']).decode(),
+		re.MULTILINE
+	)]
+
+	powerlines = []
```

Comment:
You need only *one* powerline, since themes are now selected using local_themes facility. Local theme should be selected not at powerline creation, but at render call. And also …